contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_present_in_other_table.py

```diff
@@ -0,0 +1,299 @@
+                        "success": True,
+                        "result": {
+                            "observed_value": "0 missing values.",
+                            "unexpected_index_list": [],
+                        },
+                    },
+                },
+            ],
+        },
+        {
+            "data": [
+                {
+                    "dataset_name": "order_table_2",
+                    "data": {
+                        "ORDER_ID": ["aaa", "bbb", "ccc"],
+                        "CUSTOMER_ID": [1, 5, 6],
+                    },
+                },
+                {
+                    "dataset_name": "customer_table",
+                    "data": {
+                        "CUSTOMER_ID": [1, 2, 3],
+                    },
+                },
+            ],
+            "only_for": ["snowflake", "sqlite"],
+            "tests": [
+                {
+                    "title": "basic_negative_test",
+                    "exact_match_out": False,
+                    "include_in_gallery": True,
+                    "in": {
+                        "foreign_key_column": "CUSTOMER_ID",
+                        "foreign_table": "customer_table",
+                        "foreign_table_key_column": "CUSTOMER_ID",
+                    },
+                    "out": {
+                        "success": False,
+                        "result": {
+                            "observed_value": "2 missing values.",
+                            "unexpected_count": 2,
+                            "unexpected_index_list": [
+                                {"customer_id": "5"},
+                                {"customer_id": "6"},
+                            ],
+                        },
+                    },
+                },
+            ],
+        },
+    ]
+
+    @root_validator
+    def _validate_template_dict(cls, values):
+        template_dict: dict = {
+            "foreign_key_column": values["foreign_key_column"],
+            "foreign_table": values["foreign_table"],
+            "foreign_table_key_column": values["foreign_table_key_column"],
+        }
+        values["template_dict"] = template_dict
+        return values
+
+    @classmethod
+    @override
+    @renderer(renderer_type="renderer.prescriptive")
+    def _prescriptive_renderer(
+        cls,
+        configuration: Optional[ExpectationConfiguration] = None,
+        result: Optional[ExpectationValidationResult] = None,
+        runtime_configuration: Optional[dict] = None,
+    ) -> List[RenderedStringTemplateContent]:
+        runtime_configuration = runtime_configuration or {}
+        styling = runtime_configuration.get("styling")
+
+        foreign_key_column: str = configuration.kwargs.get("foreign_key_column")
+        foreign_table: str = configuration.kwargs.get("foreign_table")
+        foreign_table_key_column: str = configuration.kwargs.get(
+            "foreign_table_key_column"
+        )
```

Comment:
Small request - we're guaranteed these values due to prior validation so can we use `[]` instead of `.get`?